operate/client/src/App/ProcessInstance/ProcessInstanceHeader/ProcessInstanceOperations.tsx

```diff
@@ -75,17 +56,30 @@ const ProcessInstanceOperations: React.FC<Props> = ({processInstance}) => {
     },
   });
 
+  const {
+    mutate: resolveProcessInstanceIncidents,
+    isPending: isResolveIncidentsPending,
+  } = useResolveProcessInstanceIncidents(processInstance.processInstanceKey, {
+    onError: (error) => {
+      invalidateQueries();
+      handleOperationError(error.status);
+    },
+    onSuccess: () => {
+      invalidateQueries();
+      tracking.track({
+        eventName: 'single-operation',
+        operationType: 'RESOLVE_INCIDENT',
+        source: 'instance-header',
+      });
+    },
+  });
+
   const invalidateQueries = () => {
     queryClient.invalidateQueries({
       queryKey: [PROCESS_INSTANCE_DEPRECATED_QUERY_KEY],
     });
   };
 
-  const handleOperationError: ErrorHandler = ({statusCode}) => {
-    invalidateQueries();
-    handleOperationErrorUtil(statusCode);
-  };
-
   const handleOperationSuccess = (operationType: OperationEntityType) => {
```

Comment:
💭 Since `handleOperationError` is getting inlined, we could also inline this function `handleOperationError`, which is only used for one callback. Alternatively, I would keep `handleOperationError` and use both callback in `useResolveProcessInstanceIncidents` as well.